CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/Context.java

```diff
@@ -8,4 +8,11 @@
     InputStream getAsset(String name);
     String getPlatform();
     String getLocalIpAddress();
+
+    /** the customEncode method allows custom Base64 package
```

Comment:
This is a JavaDoc comment (two stars).  I think you probably just want a normal multi-line comment here:
/*
 *
 */

Reply:
thank you for suggestions, made changes